curriculum/challenges/english/25-front-end-development/lecture-welcome-to-freecodecamp/6734ddabad59e593a49afafe.md

```diff
@@ -8,7 +8,7 @@ dashedName: welcome-message-from-quincy
 
 # --description--
 
-Watch the video and complete the assignment below.
+Watch the video or read the transcript and answer the questions below.
```

Comment:
Actually, it looks like this is the only video that doesn't have questions associated with it

```suggestion
Watch the video or read the transcript and complete the assignment below.
```